src/man/firejail.1.in

```diff
@@ -1246,27 +1246,27 @@ Enforce the Landlock ruleset.
 Without it, the other Landlock commands have no effect.
 See the \fBLANDLOCK\fR section for more information.
 .TP
-\fB\-\-landlock.fs.read=path (experimental)
-Create a Landlock ruleset (if it doesn't already exist) and add a read access
-rule for path.
+\fB\-\-landlock.fs.execute=path (experimental)
+Create a Landlock ruleset (if it doesn't already exist) and add an execution
+permission rule for path.
 .TP
-\fB\-\-landlock.fs.write=path (experimental)
-Create a Landlock ruleset (if it doesn't already exist) and add a write access
-rule for path.
+\fB\-\-landlock.fs.makedev=path (experimental)
+Create a Landlock ruleset (if it doesn't already exist) and add a rule that
+allows the creation of block devices and character devices beneath the given
+path.
 .TP
 \fB\-\-landlock.fs.makeipc=path (experimental)
 Create a Landlock ruleset (if it doesn't already exist) and add a rule that
 allows the creation of named pipes (FIFOs) and Unix domain sockets beneath
 the given path.
 .TP
-\fB\-\-landlock.fs.makedev=path (experimental)
-Create a Landlock ruleset (if it doesn't already exist) and add a rule that
-allows the creation of block devices and character devices beneath the given
-path.
+\fB\-\-landlock.fs.read=path (experimental)
+Create a Landlock ruleset (if it doesn't already exist) and add a read access
+rule for path.
 .TP
-\fB\-\-landlock.fs.execute=path (experimental)
-Create a Landlock ruleset (if it doesn't already exist) and add an execution
-permission rule for path.
+\fB\-\-landlock.fs.write=path (experimental)
+Create a Landlock ruleset (if it doesn't already exist) and add a write access
+rule for path.
```

Comment:
See:

* <https://github.com/netblue30/firejail/pull/6203#pullrequestreview-1873100859>

Reply:
My bad, forgot that we touched this before. Reverted the landlock re-ordering. Thanks for the review!